dfdaemon/handler/root_handler.go

```diff
@@ -18,16 +18,14 @@ package handler
 
 import (
 	"net/http"
-	"net/http/pprof"
+	// pprof will inject handlers for users to profile this program
+	_ "net/http/pprof"
 )
 
 // New returns a new handler for dfdaemon
 func New() *http.ServeMux {
-	s := http.NewServeMux()
+	s := http.DefaultServeMux
 	s.HandleFunc("/args", getArgs)
-	s.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
-	s.HandleFunc("/debug/pprof/", pprof.Index)
-	s.HandleFunc("/debug/version", getVersion)
```

Comment:
```
#!/bin/bash -eo pipefail
gometalinter --disable-all --skip vendor --skip test -E deadcode ./...
dfdaemon/handler/debug_handler.go:25:1:warning: getVersion is unused (deadcode)
```
When you remove one parameter, you have to check if this is a only place to use it. If it is, after removal, it will be deadcode. Please help to check this. Thanks.

Reply:
Thanks. I have updated it. PTAL @allencloud